front-end/src/components/Navbar.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import {Link} from 'react-router-dom'
+
+const Navbar = (props) => {
+  return (
+    <nav>
+      <Link to='/'>
+        <a>Logo/Home</a>
+      </Link>
+      <Link to='/login'>
+        <a>Login/Signup</a>
+      </Link>
+      <Link to='/logout'>
+        <a>Logout</a>
+      </Link>
+    </nav>
+  )
+}
+
+export default Navbar
```

Comment:
where are you doing the CSS for this navbar and other components?

Reply:
@lukeschunk Currently they are unstyled so we can decide on a cohesive design. @lcurvelo27 and I need to come up with that that will look like. I think most of the components (Homepage, navbar, search, editing page) will reference a single stylesheet to keep the look and feel the same. The theme components will have inline styling so that they can not interfere with the other components.

Reply:
@jfirwin that looks good to me.